contrib/endpoints/src/api_manager/config_test.cc

```diff
@@ -93,13 +93,13 @@ TEST(Config, ServerConfigProto) {
 }
 
 static const char kInvalidServerConfig[] = R"(
-service_control_config {
-  type: 1
-  config {
-    cache_entries: 1020
-    flush_interval_ms: 15
+  service_control_config {
+    type: 1
```

Comment:
Actually the raw string style is kinda new to me. And I realized the style even in this file is inconsistent. So I did some search at istio/proxy repo and google3 (https://cs.corp.google.com/search/?q=%22R%5C%22(%22+lang:cc+case:yes) and tried to follow what most people did. 

I just kept the existing style as it is. No big deal :)